eng/pipelines/templates/steps/install-latest-jdk.yml

```diff
@@ -0,0 +1,25 @@
+steps:
+  # Non-standard JDK versions are only supported in Linux.
+  # Make the assumption here Linux is being used, if it's not it's a configuration issue that needs to be fixed there.
+  - task: Cache@2
+    inputs:
+      key: 'jdk | "$(JavaTestVersion)" | "$(CacheSalt)" | "$(Agent.OS)"'
+      path: $(Agent.BuildDirectory)/jdk-19+36
+    displayName: 'Cache Latest JDK'
+    condition: eq(variables['LatestJdkVersion'], variables['JavaTestVersion'])
+
+  - task: PowerShell@2
+    displayName: 'Install Latest JDK'
+    inputs:
+      pwsh: true
+      workingDirectory: $(Agent.BuildDirectory)
+      filePath: eng/scripts/Install-Latest-JDK.ps1
```

Comment:
is the path for the cache vs. the install filepath going to line up when future changes to the script are made? i.e. should the child directory be something a little more generic so the caching doesn't break?

Reply:
Definitely not, I need to look into renaming the folder in the tar/zip after unpacking to make this more durable. I'm thinking something like `jdk-19+36` -> `jdk-19`, that way the path can cleanly become `path: $(Agent.BuildDirectory)/jdk-$(LatestJdkVersion)`, thoughts?